src/Entity.php

```diff
@@ -67,15 +67,21 @@ protected function request($method, $relativeUrl, $data = null)
         }
         else
         {
-            return static::buildEntity($response);
+            return static::buildEntity(null, $response);
```

Comment:
@captn3m0 not sure if it's a good idea to pass null. Was thinking if we can make key as the second argument. What do you think?